services/brig/test/integration/API/Provider.hs

```diff
@@ -987,7 +988,7 @@ lookupCode db gen = liftIO . DB.runClient db . Code.lookup (Code.genKey gen)
 --------------------------------------------------------------------------------
 -- Utilities
 
-randomProvider :: DB.ClientState -> Brig -> Http Provider
```

Comment:
Lots of `HasCallStack`s from this point and nothing else, keep reading until you see another comment.